src/components/header/OverflowEdgeIndicator.js

```diff
@@ -2,21 +2,40 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
 import React, { PureComponent, PropTypes } from 'react';
 import classNames from 'classnames';
 
 import './OverflowEdgeIndicator.css';
 
+type Props = {
+  className: string,
+  children: number,
+};
+
 class OverflowEdgeIndicator extends PureComponent {
-  constructor(props) {
+  props: Props;
+  state: {
+    overflowsOnTop: ?boolean,
```

Comment:
I'm pretty sure that `classNames` would accept a boolean. I'd like to see actual code producing the error to give more guidance :)